File: TypeScript/ddsk.ts
const main = () => {
  const DDSK = ["ドド", "スコ"];
  const ddsk_temp = "ドドスコスコスコドドスコスコスコドドスコスコスコ";
  let text = "";
  while (true) {
    const Random = Math.floor(Math.random() * 2);
    text += DDSK[Random];
    console.log(text);
    if (text.includes(ddsk_temp) == true) {
      console.log("ラブ注入♡");
      break;
    }
  }
};
main();

File: Dart/ddsk.dart
import 'dart:math';

void main()
{
    var DDSK = ["ドド", "スコ"];
    var ddsk_temp = "ドドスコスコスコドドスコスコスコドドスコスコスコ";
    var text = "";

    while (!text.contains(ddsk_temp)) {
        var random = Random().nextInt(2);
        print(DDSK[random]);
        text += DDSK[random];
    }

    print("ラブ注入♡");
}
